app/javascript/components/pages/NotFound.test.js

```diff
@@ -1 +1,28 @@
-import "@testing-library/jest-dom"
\ No newline at end of file
+import React from "react"
+import { screen, render } from "@testing-library/react"
+import NotFound from "./NotFound"
+import { BrowserRouter } from "react-router-dom"
+import "@testing-library/jest-dom"
+
+describe("<NotFound />", () => {
+    beforeEach(() => {
```

Comment:
This indentation looks a bit different than the rest, can we make it uniform across.